pkg/components/metallb/manifests.go

```diff
@@ -948,24 +948,32 @@ spec:
     - alert: MetalLBNoBGPSession
       expr: metallb_bgp_session_up != 1
       for: 2m
+      labels:
+        severity: critical
```

Comment:
Hm, I see 2 conventions around those labels. One is to use `page` and another to use `critical`. What consumes this label later on?

Reply:
Do you have a reference to the `page` level, because I can't find any references to it when [searching this repo](https://github.com/kinvolk/lokomotive/search?q=page&unscoped_q=page) ? When I check our current lokomotive cluster, I don't find any severity labels with `page`, only `none`, `warning` or `critical`.
`kubectl get prometheusrule --all-namespaces -oyaml | grep "page"`

note: we don't have all the components installed so could be we are missing some alert rules.

Reply:
Ah yes, I should've grep the code before asking. Sorry, I'm not familiar with alert manager conventions. But it seems prometheus-operator ships many `severity: critical`, so it seems reasonable to just follow that.